CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This changelog is a work in progress and may contain notes for versions which ha
 
 ## v6.0.2-beta
 
+### Features ✅ 
```

Comment:
Hmm... I wonder if this should be considered a bug fix instead of a feature? It's not something that end users are ever going to "use" per se.

If you do want this to be a feature, then we should bump the upcoming release to `6.1.0-beta`.

Reply:
Agreed.